src/Blocks/BlockButton.h

```diff
@@ -67,7 +67,7 @@ class cBlockButtonHandler :
 			case BLOCK_FACE_XM: return 0x2;
 			case BLOCK_FACE_XP: return 0x1;
 			case BLOCK_FACE_YM: return 0x0;
-			default:
+			case BLOCK_FACE_NONE:
```

Comment:
Is this change correct?

I think the point is to detect if there is a value not included in the enum that has been passed in.